src/dao/pages/CreateADAO/forms/MultiSigDAOVotingForm.tsx

```diff
@@ -11,8 +11,8 @@ export function MultiSigDAOVotingForm() {
     formState: { errors },
   } = useFormContext<CreateAMultiSigDAOForm>();
   const { governance } = getValues();
-  const { admin, owners } = governance;
-  const adminCount = admin.length > 0 ? 1 : 0;
+  const { admin, owners } = governance ?? { admin: "", owners: [] };
+  const adminCount = admin?.length > 0 ? 1 : 0;
```

Comment:
This change resolves one issue for `undefined admin ` in voting form when page refreshes